pkg/elemental/pages/index.vue

```diff
@@ -0,0 +1,190 @@
+<script>
+import { ELEMENTAL_SCHEMA_IDS, ELEMENTAL_CLUSTER_PROVIDER } from '../types';
+import { allHash } from '@shell/utils/promise';
+import { createElementalRoute } from '../utils/custom-routing';
+import { filterForElementalClusters } from '../utils/elemental-utils';
+import Loading from '@shell/components/Loading';
+import { CAPI } from '@shell/config/types';
+
+export default {
+  name:       'Dashboard',
+  components: { Loading },
+  async fetch() {
+    const allDispatches = await allHash({
+      machineRegistrations: this.$store.dispatch('management/findAll', { type: ELEMENTAL_SCHEMA_IDS.MACHINE_REGISTRATIONS }),
+      machineInventories:   this.$store.dispatch('management/findAll', { type: ELEMENTAL_SCHEMA_IDS.MACHINE_INVENTORIES }),
+      rancherClusters:      this.$store.dispatch('management/findAll', { type: CAPI.RANCHER_CLUSTER }),
+      managedOsImages:      this.$store.dispatch('management/findAll', { type: ELEMENTAL_SCHEMA_IDS.MANAGED_OS_IMAGES }),
+    });
+
+    this.resourcesData = {};
+
+    this.resourcesData[ELEMENTAL_SCHEMA_IDS.MACHINE_REGISTRATIONS] = allDispatches.machineRegistrations;
+    this.resourcesData[ELEMENTAL_SCHEMA_IDS.MACHINE_INVENTORIES] = allDispatches.machineInventories;
+    this.resourcesData[this.ELEMENTAL_CLUSTERS] = filterForElementalClusters(allDispatches.rancherClusters);
+    this.resourcesData[ELEMENTAL_SCHEMA_IDS.MANAGED_OS_IMAGES] = allDispatches.managedOsImages;
+  },
+  data() {
+    return { ELEMENTAL_CLUSTERS: 'elementalClusters' };
+  },
+  computed: {
+    cards() {
+      const out = [];
+
+      const clusterCreateRoute = {
+        name:   'c-cluster-product-resource-create',
+        params: {
+          resource: CAPI.RANCHER_CLUSTER,
+          product:  'manager',
+        },
+        query: { type: ELEMENTAL_CLUSTER_PROVIDER }
+      };
+
+      const clusterManageRoute = {
+        name:   'c-cluster-product-resource',
+        params: {
+          resource: CAPI.RANCHER_CLUSTER,
+          product:  'manager',
+        }
+      };
+
+      [ELEMENTAL_SCHEMA_IDS.MACHINE_REGISTRATIONS,
```

Comment:
Just a readability thing, the below is a bit clearer

```
      [
        ELEMENTAL_SCHEMA_IDS.MACHINE_REGISTRATIONS,
        ELEMENTAL_SCHEMA_IDS.MACHINE_INVENTORIES,
        this.ELEMENTAL_CLUSTERS,
        ELEMENTAL_SCHEMA_IDS.MANAGED_OS_IMAGES
      ].forEach((type) => {
```

Reply:
agree 👍  changed